percy-mirror-verifier/test/github.test.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const mockOctokit = {
+  pulls: {
+    get: jest.fn(),
+  },
+  repos: {
+    listCommitStatusesForRef: jest.fn(),
+  },
+};
+jest.doMock('@octokit/rest', () => ({
+  Octokit: jest.fn().mockImplementation(() => mockOctokit),
+}));
+
+import {RestEndpointMethodTypes} from '@octokit/rest';
+
+import {getPercyBuildId} from '../src/github';
+
+describe('github', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('getPercyBuildId', () => {
+    it('gets Percy build id', async () => {
+      mockOctokit.pulls.get.mockResolvedValue({
+        data: {head: {sha: '6f91e57c4ec735bda983f253304b46058518f62a'}},
+      } as RestEndpointMethodTypes['pulls']['get']['response']);
```

Comment:
Nonblocking: It might be worth extracting this at the top of the file.
`type OctokiPullResponse = RestEndpointMethodTypes['pulls']['get']['response'];`

Reply:
Done 0af5d1f